### PR TITLE
fix(anchor):[anchor]When the anchor component has a default currentLink, the style is not the expected style

### PR DESCRIPTION
**Pull request by fanbingbing16**
…nk, the style is not the expected style

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #1366 

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by chenxi-20** [comment]
这个是什么使用场景下会出现的问题，还请给出复现步骤~ @fanbingbing16

**Review by chenxi-20 — COMMENTED** [review]
1

**Review by chenxi-20 — CHANGES_REQUESTED** [review]
1

File: packages/renderless/src/anchor/index.ts [hunk 1]
```diff
@@ -165,6 +165,7 @@ export const onItersectionObserver =
   ({ state, props, api, vm, emit }: Pick<IAnchorRenderlessParams, 'state' | 'props' | 'api' | 'vm' | 'emit'>) =>
   () => {
     const { expandLink, scrollContainer } = state
+	state.currentLink && updateSkidPosition({ vm, state, emit })
```
Comment (chenxi-20) [inline packages/renderless/src/anchor/index.ts]: 这个是什么使用场景下会出现的问题，还请给出复现步骤,，并建议格式化一下~ @fanbingbing16

**Review by chenxi-20 — DISMISSED** [review]

File: packages/renderless/src/anchor/index.ts (hunk 1 above)
Comment (fanbingbing16) [reply packages/renderless/src/anchor/index.ts]: Issue Number: https://github.com/opentiny/tiny-vue/issues/1366  可以查看这个issue

**Review by chenxi-20 — DISMISSED** [review]

**Review by chenxi-20 — COMMENTED** [review]
好的，麻烦格式化一下代码，缩进不太对~

**Comment by fanbingbing16** [comment]
@chenxi-20 改好了

**Review by chenxi-20 — APPROVED** [review]

**Review by zzcr — APPROVED** [review]

**Pull request merged**